apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -77,41 +78,42 @@ const configuredUiItems = new Map<string, UiItem>([
         await TreeWidget.initialize();
       },
       createUiItemsProviders: () => [
-        new TreeWidgetUiItemsProvider({
+        new TreeWidgetWithOptionsUiItemsProvider({
```

Comment:
I think this can be simplified using new UIProvider API:
```tsx
createUIItemsProviders: () => [{
  id: "TreeWidgetUIProvider,
  getWidgets: () => {
    const trees = [ // tree definitions ]
    return [{
      id: "tree-widget",
      content: <TreeWidgetWithOptions trees={trees} />,
      layouts: {
        standard: {
           section: StagePanelSection.Start,
           location: StagePanelLocation.Right,
        }
      }
    }]
}]
```

Then you can move `TreeWidgetWithOptions` to this file and remove `TreeWidgetWithOptionsUiItemsProvider.tsx`